src/index.ts

```diff
@@ -1 +1 @@
-export {run} from '@oclif/command'
+export { run } from "@oclif/command";
```

Comment:
do we have prettier set for this? If not we should (typically spaces before curlies and single quotes

Reply:
yeah, this is a global prettier config I'm using - we don't have one on the project yet.